condor_watch_q.py

```diff
@@ -988,6 +996,15 @@ def make_table(headers, rows, fill="", header_fmt=None, row_fmt=None, alignment=
         for original_row, processed_row in zip(rows, processed_rows)
     ]
 
+    remaining_columns = terminal_columns - len(lines[0])
```

Comment:
I suspect we need to be careful with ANSI handling here, given the other things in the pipeline. Maybe `len(strip_ansi(lines[0]))`?